vcpkg/get_started/get-started-vscode.md

```diff
@@ -21,211 +21,216 @@ dependencies, configure, build, and run a simple application.
 - [C++ compiler](https://code.visualstudio.com/docs/languages/cpp#_install-a-compiler)
 - Windows 7 or newer
 
-This example uses the C++ MSVC compiler in the Visual Studio C++ development workload.
-
 ## 1 - Set up vcpkg
 
 [!INCLUDE [setup-vcpkg](includes/setup-vcpkg.md)]
 
-## 2 - Install Visual Studio Code Extensions
+## 2 - Create a project folder
+
+  > [!NOTE]
+  > If you're running this tutorial in Windows and using MSVC as your compiler
+  > you need to start the Visual Studio Code instance from a Developer Command
+  > Prompt for VS or Developer PowerShell for VS. This ensures that the compiler
+  > path and other environment variables are correctly set up.
```

Comment:
On Windows, doesn't vcpkg do the compiler/toolset detection for MSVC? I'm not sure if starting a VS Code instance from the developer command prompt is needed unless you wanted to run `cl.exe`.

Reply:
That's true when using the `Visual Studio` generator. We use Ninja to keep consistency between examples, which means we do one of the following:

1) Add this note and use Ninja
2) Change this tutorial to use the Visual Studio generator instead

I would like to keep using Ninja since that lets us have the same presets for Windows and non-Windows tutorials.